tox.ini

```diff
@@ -1,34 +1,32 @@
 [tox]
-envlist = linters, py3, integration
+envlist = linters, unit
 isolated_build = True
 
 [testenv]
+description = Run all tests with {basepython}
 usedevelop = True
-setenv =
-  VIRTUAL_ENV={envdir}
 deps =
-    pytest
-commands =
-    py.test -v test -k "not integration" {posargs}
+    -r {toxinidir}/test/requirements.txt
+commands = pytest {posargs:test}
```

Comment:
Instead of setting "test" here, it's usually better to set it in `test_paths =` in `pytest.ini`

Reply:
Does that let you override it with positional args?

Reply:
Yep